umbraco/Umbraco-CMS
# Packages will export Doctype & Property UDI Keys

Fixes https://github.com/umbraco/Umbraco-CMS/issues/8918

## Problem
The underlying problem was that the package.xml that is generated was not outputting the Unique Guid/Key (UDI) for each doctype. So when another Umbraco site/environment imported these doctypes they had no gurantee of using the exact same key.

In scenarios such as the new block editor. It's datatype prevalues stored a reference to the doctype (element types) it can use as a reference to it by it's key. So when the next Umbraco installed the doctypes it would not correctly link the element types to the block editor.

## Test Notes
* Build PR & run site from SLN
* Create a doctype that uses a block editor with two or more element types
* Create a local package & save it to file system
* Use the build powershell to create ZIP build & run site with VSCode & IISExpress
* Unzip the ZIP & look at package.xml & verify the new <Key> XML elements for doctypes & its properties
* Install package from new zip generated
* Verify DataType configuration of Block Editor has correctly linked to the element types

## Thought
Should more of the items in package.xml use their keys ?
